app/views/sites/contacts/_edit_form.html.erb

```diff
@@ -1,49 +1,47 @@
+<%= form_for [@site, contact] do |form| %>
+  <% if contact.errors.any? %>
+    <div id="error_explanation" class="bg-red-50 text-red-500 px-3 py-2 font-medium rounded-lg mt-3">
+      <h2><%= pluralize(contact.errors.count, "error") %> prohibited this contact from being saved:</h2>
+      <ul>
+        <% contact.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <div>
+    <div class="my-5">
+      <%= form.label :title, class: "fancy_label" %>
+      <%= form.text_field :title, required: true, placeholder: "Title", class: "input_text_field" %>
+    </div>
 
-  <%= form_for [@site, contact] do |form| %>
-    <% if contact.errors.any? %>
-      <div id="error_explanation" class="bg-red-50 text-red-500 px-3 py-2 font-medium rounded-lg mt-3">
-        <h2><%= pluralize(contact.errors.count, "error") %> prohibited this contact from being saved:</h2>
-
-        <ul>
-          <% contact.errors.each do |error| %>
-            <li><%= error.full_message %></li>
-          <% end %>
-        </ul>
-      </div>
-    <% end %>
-    <div>
-      <div class="my-5">
-        <%= form.label :title, class: "fancy_label" %>
-        <%= form.text_field :title, required: true, placeholder: "Title", class: "input_text_field" %>
-      </div>
-
-      <div class="my-5">
-        <%= form.label :first_name, class: "fancy_label" %>
-        <%= form.text_field :first_name, required: true, placeholder: "First Name", class: "input_text_field" %>
-      </div>
-
-      <div class="my-5">
-        <%= form.label :last_name, class: "fancy_label" %>
-        <%= form.text_field :last_name, required: true, placeholder: "Last Name", class: "input_text_field" %>
-      </div>
+    <div class="my-5">
```

Comment:
Please consider removing blank line between all the **_div_** tags

Reply:
I would rather keep these blank lines. All _form files have blank lines between all `<div>field description</div>`.
It's much easier to read _form files with blank line dividers between fields.
_form files were created with this format by Rails: `bin/rails g scaffold ...`